airbyte-integrations/connectors/destination-snowflake/src/main/kotlin/io/airbyte/integrations/destination/snowflake/SnowflakeDestination.kt

```diff
@@ -52,38 +63,108 @@ constructor(
     private val airbyteEnvironment: String,
     private val nameTransformer: NamingConventionTransformer = SnowflakeSQLNameTransformer(),
 ) : BaseConnector(), Destination {
+    private val destinationColumns = JavaBaseConstants.DestinationColumns.V2_WITHOUT_META
 
     override fun check(config: JsonNode): AirbyteConnectionStatus? {
-        val snowflakeInternalStagingSqlOperations =
-            SnowflakeStagingClient(nameTransformer)
         val dataSource = getDataSource(config)
         try {
+            val retentionPeriodDays = 1
```

Comment:
this is _really_ ugly... I understand we want to move to a place where the check actually is running a sync, but I don't think this is the right way to d that. I'd rather keep the old way TBH.